outbound/config.js

```diff
@@ -42,6 +42,9 @@ function load_config () {
     if (!cfg.received_header) {
         cfg.received_header = config.get('outbound.received_header') || 'Haraka outbound';
     }
+    if (!cfg.local_mx_ok) {
```

Comment:
I think this needs added to `booleans` at the top too.

Reply:
Thanks, fixed. :)